test/extended/authorization/rbac/groups_default_rules.go

```diff
@@ -116,7 +116,7 @@ var (
 
 			// These custom resources are used to extend console functionality
 			// The console team is working on eliminating this exception in the near future
-			rbacv1helpers.NewRule(read...).Groups(consoleGroup).Resources("consoleclidownloads", "consolelinks", "consolenotifications").RuleOrDie(),
+			rbacv1helpers.NewRule(read...).Groups(consoleGroup).Resources("consoleclidownloads", "consolelinks", "consoleexternalloglinks", "consolenotifications").RuleOrDie(),
```

Comment:
@spadgett @rhamilto @jhadvig currently on `consoleexternalloglinks`. I believe we are settled on this.